fili-generic-example/README.md

```diff
@@ -0,0 +1,83 @@
+Fili Generic Loader Application
+==================================
+
+This is application is meant to be able to run and connect to any instance of Druid to show the basic metrics and dimensions.
```

Comment:
😄 To answer this question, the SystemConfig component _should_ pick up these parameters as something like environment variables, so we shouldn't need to do anything special. If I'm wrong, we should loop @michael-mclawhorn in, since he's most familiar with the SystemConfig system.